client/src/components/TrackList.vue

```diff
@@ -51,7 +51,13 @@ export default Vue.extend({
   }),
 
   computed: {
-    virtualListItems() {
+    virtualListItems(): {
+      trackId: number;
+      selectedTrackId: number;
+      checkedTrackIds: number[];
+      editingTrack: boolean;
+      allTypes: string[];
+    }[] {
```

Comment:
Are the changes in this file unrelated?

Reply:
Yeah.   There's some dumb stuff you have to do to get typescript to work in Object API components.  

This block of code was missing, which is why I had to `// @ts-ignore` below.